content/cli/workspaces.md

```diff
@@ -283,29 +284,34 @@ module.exports = function(options) {
     ...options,
     externals: [],
   };
-}
+};
 ```
 
 #### Assets
 
-TypeScript compilation automatically distributes compiler output (`.js` and `.d.ts` files) to the specified output directory.  It can also be convenient to distribute non-TypeScript files, such as `.graphql` files, `images`, `.html` files and other assets.  This allows you to treat `nest build` (and any initial compilation step) as a lightweight **development build** step, where you may be editing non-TypeScript files and iteratively compiling and testing.
+TypeScript compilation automatically distributes compiler output (`.js` and `.d.ts` files) to the specified output directory. It can also be convenient to distribute non-TypeScript files, such as `.graphql` files, `images`, `.html` files and other assets. This allows you to treat `nest build` (and any initial compilation step) as a lightweight **development build** step, where you may be editing non-TypeScript files and iteratively compiling and testing.
 
-The value of the `assets` key should be an array of elements specifying the files to be distributed.  The elements can be simple strings with `glob`-like file specs, for example:
+The value of the `assets` key should be an array of elements specifying the files to be distributed. The elements can be simple strings with `glob`-like file specs, for example:
 
 ```typescript
-"assets": ["**/*.graphql"]
+"assets": ["**/*.graphql"],
+"watchAssets": true,
 ```
 
+> info **Hint** To watch your assets during development, use `watchAssets` option
+
 For finer control, the elements can be objects with the following keys:
+
 - `"include"`: `glob`-like file specifications for the assets to be distributed
 - `"exclude"`: `glob`-like file specifications for assets to be **excluded** from the `include` list
-- `"outDir"`: a string specifying the path (relative to the root folder) where the assets should be distributed.  Defaults to the same output directory configured for compiler output.
+- `"outDir"`: a string specifying the path (relative to the root folder) where the assets should be distributed. Defaults to the same output directory configured for compiler output.
+- `"watchAssets"`: accept boolean, watch specific assets from the `include` list
```

Comment:
suggestion:
accept boolean, watch specific assets from the `include` list
->
boolean; if true, run in watch mode watching specified assets

Reply:
Will be done.